mobilecoind-dev-faucet/src/worker.rs

```diff
@@ -1012,6 +1039,7 @@ impl WorkerTokenState {
     // Check if a key image is part of an in-flight transaction
     fn key_image_is_in_flight(&self, key_image: &KeyImage) -> bool {
         self.in_flight_split_tx_states.contains_key(key_image)
+            || self.in_flight_rebalancing_key_images.contains(key_image)
```

Comment:
i added this also to try to avoid (and added the self.in_flight_rebalancing_key_images variable) to try to reduce the "key image already spent" errors. I'm not sure it will really fix it but worth trying